packages/office-ui-fabric-react/src/utilities/selection/SelectionZone.test.tsx

```diff
@@ -49,7 +49,7 @@ function _initializeSelection(selectionMode = SelectionMode.multiple): void {
         <button id='toggle1' data-selection-toggle={ true }>Toggle</button>
         <button id='invoke1' data-selection-invoke={ true }>Invoke</button>
         <button id='noSelect1'>No Select</button>
-        <button id='select1' data-selection-select={ true }>Select First</button>
+        <button id='select1' data-selection-auto-select={ true }>Select First</button>
```

Comment:
this is too verbose. Why this change?

Reply:
I also think `selection-select` is redundant with no obvious reasoning, though